src/store/api/weatherProviders/accuweather.js

```diff
@@ -0,0 +1,100 @@
+import * as utils from './weatherutils';
+import getAPIData from '../apiutils';
+
+export const getCurrentPositionFromAPI = async state => {
+  const getCurrentPositionAPIUrl = ({ latitude, longitude, APIkey }) =>
+    `https://dataservice.accuweather.com/locations/v1/cities/geoposition/search?apikey=${APIkey}&q=${latitude},${longitude}&language=uk-ua`;
+
+  const currentPositionAPIUrl = getCurrentPositionAPIUrl({
+    latitude: state.currentPosition.latitude,
+    longitude: state.currentPosition.longitude,
+    APIkey: process.env.VUE_APP_ACCUWEATHER_KEY,
+  });
+  const positionJson = await getAPIData(currentPositionAPIUrl).catch(e => {
+    throw e;
+  });
+  if (positionJson.error) return positionJson;
+  if (positionJson.Key) {
+    return {
+      Key: positionJson.Key,
+      City: positionJson.LocalizedName,
+      dataLoadedFromAPI: true,
+    };
+  }
+  throw new Error('Помилка при отриманні поточної погоди');
```

Comment:
throw this error in line 14 instead and remove if in the line 17

Reply:
These errors are different. One for errors during http request and another for error when program gets response without valid data.